shim/opentelemetry-opentracing-shim/src/opentelemetry/shim/opentracing_shim/__init__.py

```diff
@@ -684,7 +684,7 @@ def start_span(
         context = SpanContextShim(span.get_span_context())
         return SpanShim(self, context, span)
 
-    def inject(self, span_context, format: object, carrier: object):
+    def inject(self, span_context, formats: object, carrier: object):
```

Comment:
Not sure we can rename function parameters because a linter rule does not like it :)

Reply:
Yes, in this case, I think we can just ignore it.